src/hooks/useWebUSB.ts

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react'
+
+const LEDGER_VENDOR_ID = 0x2c97
```

Comment:
Either move this to a better home or remove this - that's not a webUSB-generic hook